go/common/viewingkey/viewing_key.go

```diff
@@ -97,6 +97,14 @@ func Sign(userPrivKey *ecdsa.PrivateKey, vkPubKey []byte) ([]byte, error) {
 
 // GenerateSignMessage creates the message to be signed
 // vkPubKey is expected to be a []byte("0x....") to create the signing message
+// todo (@ziga) Remove this method once old WE endpoints are removed
```

Comment:
Let's add a github issue lest we forget :)

Reply:
added: https://github.com/orgs/obscuronet/projects/1/views/36?pane=issue&itemId=34873032